src/components/checkbox/Checkbox.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { fireEvent, screen } from '@testing-library/react';
+
+import Checkbox from './Checkbox';
+
+import customRender from '@/test/customRender';
+
+describe('checkbox', () => {
+  it('클릭할 때마다 onCheck와 onUncheck 번갈아 동작해야 한다', () => {
+    const onCheckMock = jest.fn();
+    const onUncheckMock = jest.fn();
+    customRender(<Checkbox onCheck={onCheckMock} onUncheck={onUncheckMock} />);
+
+    fireEvent.click(screen.getByTestId('checkbox'));
+    expect(onCheckMock).toBeCalledTimes(1);
+    onCheckMock.mockClear();
+
+    fireEvent.click(screen.getByTestId('checkbox'));
+    expect(onUncheckMock).toBeCalledTimes(1);
+
+    fireEvent.click(screen.getByTestId('checkbox'));
+    expect(onCheckMock).toBeCalledTimes(1);
```

Comment:
```suggestion
    fireEvent.click(screen.getByTestId('checkbox'));
    expect(onUncheckMock).toBeCalledTimes(1);

    fireEvent.click(screen.getByTestId('checkbox'));
    expect(onCheckMock).toBeCalledTimes(2);
```

이렇게 작성하는 게 조금 더 사용자 흐름에 가까울 거 같은 생각이에요 !

> 단순히 제 생각이니 무시하셔도 좋아요

Reply:
호출된 횟수를 마지막에 한꺼번에 카운트 하는거라고 이해하면 될까요??

Reply:
> 이렇게 작성하는 게 조금 더 사용자 흐름에 가까울 거 같은 생각이에요 !
> 
> > 단순히 제 생각이니 무시하셔도 좋아요

호출된 횟수를 마지막에 한꺼번에 카운트 하는거라고 이해하면 될까요??

Reply:
두 번 호출된 걸 확인한다 ... 정도로 저는 이해하고 있어요!